qiita_db/test/test_metadata_template.py

```diff
@@ -2451,6 +2458,17 @@ def test_invalid_lat_long(self):
     "2.Sample3\t2014-05-29 12:24:51\tTest Sample 3\tTrue\tTrue\tNotIdentified"
     "\t3\t4.8\t4.41\tlocation1\treceived\ttype1\tValue for sample 3\n")
 
+EXP_SAMPLE_TEMPLATE_MULTICASE = (
+    "Sample_naMe\tcoLLection_timestamp\tdesCription\thas_exTracted_data\t"
```

Comment:
It would be great to have another tests that guaranteed that non-required columns remain with the casing unaltered.